internal/runbits/requirements/requirements.go

```diff
@@ -431,8 +517,19 @@ func (r *RequirementOperation) updateCommitID(commitID strfmt.UUID) error {
 	return nil
 }
 
-func (r *RequirementOperation) shouldPromptForSecurity(vulnerabilities *model.Vulnerabilites) bool {
-	if !r.Config.GetBool(constants.SecurityPromptConfig) || vulnerabilities == nil {
+func (r *RequirementOperation) shouldPromptForSecurity(vulnerabilities model.VulnerableIngredientsByLevels) bool {
+	if !r.Config.IsSet(constants.SecurityPromptConfig) {
```

Comment:
Both this, and the prompt level are now registered with defaults so I don't believe we need to be setting these values in the config database for the user.